src/components/Input/Input.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function Input({ labelText, placeholder, name }) {
+  return (
+    <Label className="Input">
+      {labelText}
+      <Textarea
+        name={name}
+        placeholder={placeholder}
+        type="text"
```

Comment:
My thought was that a description of a meal is not always short and I wanted to provide more space with the textarea. But since the textarea has a limit of characters anyway, I could also stay with the input.